### ISSUE TITLE
Message Component Accessibility - Color on Color is bad for accessibility, e.g blue text on blue background

### ISSUE THREAD

**Issue by dcb99** [primefaces/primevue#7040]
### Describe the bug

<img width="820" alt="image" src="https://github.com/user-attachments/assets/97dde0e6-5b0c-46e7-9dd4-d081876c05f3" />
This is one example, but blue text on a blue background does not conform to accessibility guidelines. It doesn't provide enough contrast and is a bad UX for those with colorblindness. 

### Reason for not contributing a PR

- [ ] Lack of time
- [X] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-39m9a5yj?file=src%2FApp.vue

### Environment

Any

### Vue version

3.5.x

### PrimeVue version

4.x

### Steps to reproduce the behavior

Using the message component, the text color is to close to the background color. 

### Expected behavior

Accessibility guidelines should be followed. Text should should be contrasted from the background.

**Comment by cagataycivici** [comment]
This depends on the theme used, each preset has a different style, you can easily customize it with the design tokens.

**Comment by dcb99** [comment]
Thanks for the response, but I don't think it's best practice to release a feature that doesn't meet accessibility requirements and then say, "fix it yourself by customizing". Customizing a theme should be used to satisfy business requirements, not to fix accessibility issues.
